tools/EntityFramework.settings.targets

```diff
@@ -24,7 +24,8 @@ Project global pre-settings.
     <VersionReleasePrefix>0</VersionReleasePrefix>
     <VersionRelease>$(VersionReleasePrefix)-beta2</VersionRelease>
     <VersionReleaseName>Beta 2</VersionReleaseName>
-    <ToolingDotNetFrameworkVersion Condition="'$(VisualStudioVersion)' == '11.0' OR '$(VisualStudioVersion)' == '12.0' OR '$(VisualStudioVersion)' == '14.0'">v4.5</ToolingDotNetFrameworkVersion>
+    <ToolingDotNetFrameworkVersion Condition="'$(VisualStudioVersion)' == '11.0' OR '$(VisualStudioVersion)' == '12.0'">v4.5</ToolingDotNetFrameworkVersion>
+    <ToolingDotNetFrameworkVersion Condition="'$(VisualStudioVersion)' == '14.0'">4.5.1</ToolingDotNetFrameworkVersion>
```

Comment:
*Running* on 4.5.1 may not be supported, but I think it's safest to *compile* against it here. On a clean install of VS 2015, they may only have 4.5.1, and I wouldn't want to prevent it from working just because it's unsupported. (My two cents.)